### PR TITLE
Use uv for images

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -64,3 +64,4 @@ streaming_prefix:
   mpeg: 'mp4'
   m4a: 'mp4'
   mp3: 'mp3'
+image_viewer: Embed::Viewer::ImageX
```
Comment (ndushay) [inline config/settings.yml]: @mejackreed  could you please add a comment and indicate what the two options are?

File: .hound.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-javascript:
+jshint:
```
Comment (ndushay) [inline .hound.yml]: this is awesome if it will shut up a lot of the hound messages.

Comment (mejackreed) [inline .hound.yml]: 🙏

File: config/settings.yml (hunk 1 above)
Comment (mejackreed) [inline config/settings.yml]: Added.. thanks for that. Great catch

**Pull request merged**